v2/internal/controllers/crd_servicebus_namespace_basic_v1api20210101preview_test.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	servicebus "github.com/Azure/azure-service-operator/v2/api/servicebus/v1api20210101preview"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_ServiceBus_Namespace_Basic_v1api20210101preview_CRUD(t *testing.T) {
```

Comment:
Going forward I wonder if we really need to clone the tests for each version or if just a single (or 2?) versions is sufficient?

It would make adding new versions easier and we have the samples recordings for the other versions so we'll still know they work.

Reply:
Definitely a conversation worth having post the v2.3.0 release.